cloud-formation/membership-app.cf.yaml

```diff
@@ -106,14 +124,12 @@ Resources:
         Fn::Base64:
           !Sub |
             #!/bin/bash -ev
-            CONF_DIR=/etc/frontend
-            aws --region ${AWS::Region} s3 cp s3://membership-dist/${Stack}/${Stage}/frontend/frontend_1.0-SNAPSHOT_all.deb /tmp
-            mkdir /etc/gu
-            aws --region ${AWS::Region} s3 cp s3://membership-private/${Stage}/membership.private.conf /etc/gu
-            dpkg -i /tmp/frontend_1.0-SNAPSHOT_all.deb
-            chown frontend /etc/gu/membership.private.conf
-            chmod 0600 /etc/gu/membership.private.conf
-            /opt/cloudwatch-logs/configure-logs application ${Stack} ${Stage} ${App} /var/log/frontend/frontend.log
+            export stack='${Stack}'
+            export app='${App}'
+            export stage='${Stage}'
+            export region='${AWS::Region}'
+            aws s3 cp --region ${AWS::Region} s3://membership-dist/${Stack}/${Stage}/frontend/init-instance.sh .
+            . init-instance.sh
```

Comment:
all the stuff is moved to a script to prevent having to cloudform when we want to change the startup of instances